hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMRecoverLeaseRequest.java

```diff
@@ -212,8 +211,7 @@ private String doWork(OzoneManager ozoneManager, long transactionLogIndex)
       LOG.warn("Key:" + keyName + " is already closed");
       return null;
     }
-    String openFileDBKey = omMetadataManager.getOpenFileName(
-            volumeId, bucketId, parentID, fileName, Long.parseLong(clientId));
+    String openFileDBKey = getKey.getOpenFileName();
```

Comment:
Correct me if I'm wrong, but since `clientID` is not set in `OmGetKey` class, the `openFileDBKey` will take null as the `clientID`?

You can refer to the earlier review on passing the `clientID` instead of storing it in the `OmGetKey` class. 

Also you can refer to https://github.com/ivandika3/ozone/commit/4803bb27faa4b7790763d9952520fa62fd057d39#diff-4a6940e76f86d73b1112150209e4f32b9fb9c08aa5b9557e75ccb31b671ceafd (still a rough work).

Reply:
Thank you for the update.